doc/developers/bug_triaging.rst

```diff
@@ -0,0 +1,146 @@
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/scikit-learn/scikit-learn/issues>`_
+is important to the communication in the project: it helps
+developers identify major projects to work on, as well as to discuss
+priorities. For this reason, it is important to curate it, adding labels
+to issues and closing issues that are not necessary.
+
+Working on issues to improve them
+--------------------------------------
+
+Improving issues increases their chances of being successfully resolved.
+Guidelines on submitting good issues can be found :ref:`here
+<filing_bugs>`. A third party can give useful feedback or even add
+comments on the issue, while core-developpers or members of the triage
+team can edit the issue description and title.
```

Comment:
The triage team as defined by github (see [here](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)) cannot edit descriptions and titles.